### PR TITLE
chore: Added Gravestone game objects

### PR DESCRIPTION
**Pull request by 0xDeon**
Summary
- Duplicated gravestone gameobject
- Repositioned all duplicates accordingly

### CODE REVIEW

**Comment by 0xDeon** [comment]
@jimenezz22 All done
Please review

**Review by jimenezz22 — COMMENTED** [review]
@DanielEmmanuel1 Can you provide evidence of the changes you made and a detailed summary of what you did? I see you are not following our pull request template

**Comment by 0xDeon** [comment]
Changes i made
- I imported the reference image into unity
- Then I drop it as the first child of the Grid gameobject making sure it perfectly fits the grid
- Then I disabled the Ground object so I can see the reference image in the background
- Then I duplicated the Grave objects and positioned them over the graves in the reference image
- Then I deleted the reference image and set the re-enabled the Ground object
- Then I deleted the reference image from the project

@jimenezz22 I didn't read the readme indepthly, that was an oversight from my part and i do apologise for that, I just went through the readme and if need be, i can still create a branch, push from there and create a new PR, whatever works for u.

Here is a screen recording
https://drive.google.com/file/d/1WhyBD6MopDopqKW-RjApwXMr_ZpHZOys/view?usp=sharing

File: Assets/Scenes/DarkForest.unity
Comment (jimenezz22) [inline Assets/Scenes/DarkForest.unity:1]: This looks like you removed the scene from the pr

When I ran your branch on my local repo, the DarkForest Scene doesn't exists
Screenshot 2024-11-26 at 2 56 17 PM

**Review by jimenezz22 — CHANGES_REQUESTED** [review]

File: Assets/Scenes/DarkForest.unity
Comment (0xDeon) [reply Assets/Scenes/DarkForest.unity:1]: I'll handle this now

**Comment by 0xDeon** [comment]
Hi @jimenezz22 

I think the issue here is someone else made updates to the scene while I was making changes to it too
And now there's a conflict because as u know GitHub cannot merge Unity scenes.

A feasible solution will be to discard/revert my changes, then I'll pull and implement the changes again
But you have to let me know as soon as you have reverted the changes to avoid another scene conflict

Is this okay?

**Comment by 0xDeon** [comment]
@jimenezz22 Please can u check now

**Review by jimenezz22 — APPROVED** [review]
pr already tested on my local repo and works successfully

**Pull request closed without merging**